package.json

```
@@ -70,6 +70,8 @@
     "http-proxy-agent": "^7.0.2",
     "https-proxy-agent": "^7.0.6",
     "lodash": "4.17.23",
```

Biome (2.4.10)

[error] The key lodash was already declared.

(lint/suspicious/noDuplicateObjectKeys)